docs/dev.md

```diff
@@ -164,3 +164,80 @@ commands results in the **LOSS OF DATA**.
     ```shell
     docker-compose down --rmi all -v
     ```
+
+## Checking if there's any documentation with invalid links
+
+The repository comes with documentation spread out over multiple MarkDown files.
+[This workflow file](./../.github/workflows/markdown-link-checker.yml) is set up to check whether there are broken links in
+any of them. If you want to check this locally, you can do so with:
+
+```shell
+# from the repository root directory
+npm install
+npm run mlc
+```
+
+## Running the superlinter locally
+
+We use GitHub's [Super-Linter](https://github.com/github/super-linter) to lint all directories using a variety of
+linters. You can run the superlinter using `docker`, as follows:
+
+```shell
+# get the linter
+docker pull github/super-linter:latest
+
+# run the linter
+docker run -e RUN_LOCAL=true \
+   -v ${PWD}/harvesting/app.py:/tmp/lint/app.py \
```

Comment:
The surrounding text suggest you want to run on whole repo here not a single file
```suggestion
   -v ${PWD}:/tmp/lint \
```

Reply:
The linter writes `super-linter.log` in PWD as root.  To prevent log file from being written as root in PWD we could use read only mount
```suggestion
   -v ${PWD}:/tmp/lint:ro \
```
Or we can add it to `/.gitignore`, its already ignored in `./harvesting/.gitignore:*.log`.

Reply:
Changed to using readonly mount